src/opower/opower.py

```diff
@@ -309,6 +317,31 @@ async def _async_get_customers(self) -> list[Any]:
         assert self.customers
         return self.customers
 
+    async def _async_get_user_accounts(self) -> list[Any]:

```

Comment:
PG&E now fails with:
`aiohttp.client_exceptions.ClientResponseError: 401, message='Unauthorized', url=URL('https://pge.opower.com/ei/edge/apis/dss-invite-v1/cws/v1/utilities/connectedaccounts?pageOffset=0&pageLimit=100')`

Reply:
Sorry about that. Added check to run this for DSS type of utilities account only. I shouldn't affect any other anymore